clients/include/rocblas.hpp

```diff
@@ -1273,6 +1273,111 @@ static constexpr auto rocblas_trsm<float> = rocblas_strsm;
 template <>
 static constexpr auto rocblas_trsm<double> = rocblas_dtrsm;
 
+// trsm_ex
```

Comment:
Looks like you forgot to handle these comments.

Reply:
Removed in 9d378b0. Don't need them since the _ex functions don't have template specializations.